### DISCUSSION TITLE
Edit user information

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2473] [Unsupported Versions] [answered]
The 'Users' section of the SOC has columns for First Name, Last Name, and Actions.  How do we edit those?  I've looked through the docs and the options available through the so-user command but I'm not seeing anything for those fields.  What is the user file that can be edited for the SOC users?  An 'Add/Edit' option directly in the interface would be great too.

Perhaps this should be defined in the docs?

**Comment by jertel** [comment, marked as answer]
The Edit capability for those fields is not enabled. There is more work to do in the area of authorization before that function can be enabled. Sorry for the confusion.
